include/librealsense2/h/rs_option.h

```diff
@@ -204,6 +205,15 @@ extern "C" {
     } rs2_host_perf_mode;
     const char* rs2_host_perf_mode_to_string( rs2_host_perf_mode perf );
 
+    /** \brief values for RS2_EMITTER_FREQUENCY option. */
+    typedef enum rs2_emitter_frequency_mode
+    {
+        RS2_EMITTER_FREQUENCY_57_KHZ = 0,  /**< Emitter frequency shall be 57 [KHZ] */
+        RS2_EMITTER_FREQUENCY_91_KHZ = 1,  /**< Emitter frequency shall be 91 [KHZ] */
```

Comment:
Do we really want the values to be hardcoded in the enum that is exposed as API? If these frequency change, we will have to modify the API. Maybe we can call them LOW and HIGH FREQUENCY?

Reply:
The origin request were made because the user had other sensors that interfere with 57HZ frequency.
CDE did some work and found a higher frequency that will also work as expected which is 91HZ.
The user cares about the frequency value itself and the API is the only place he know what he can choose.

Reply:
ok